dallinger/recruiters.py

```diff
@@ -872,6 +903,42 @@ def external_submission_url(self):
         self.prolificservice.log_response(response)
         return response
 
+    def screen_out_allowed(self, participants: list[Participant]):
+        """
+        Check which participants in a list of participants can be screened out.
+
+        Returns a list of participants that can be screened out.
+        """
+        study = self.prolificservice.get_study(self.current_study_id)
+
+        if not study.get("is_custom_screening", False):
+            raise ProlificRecruiterException(
+                f"Prolific study (ID {self.current_study_id}) doesn't allow screening-out of participants"
+            )
+
+        # Minimum wage thresholds: https://researcher-help.prolific.com/en/article/2273bd
+        prolific_min_wage_per_hour = 6
+        min_required_reward = (
+            median_time_spent_in_hours(participants) * prolific_min_wage_per_hour
+        )
+        screen_out_participants = []
+        for participant in participants:
+            if participant.status != "working":
+                continue
+
+            reward = (
+                participant.base_pay + participant.bonus
+            )  # TODO: Check if this is correct
```

Comment:
It's not correct, you should be referring to `bonus_per_submission`, which needs to be passed from `_req`

Reply:
This should be fixed, too.